Index: test
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>test\r\n ,<fqsn,kfd<nklvdsnklsd<vnklvd<sknlvd<snklvdsnklvv sqnccklndsqnklcnk
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test b/test
--- a/test	(revision 32bc9f5a2dcbdaf989e57f9726c905efc30455bb)
+++ b/test	(date 1687514377698)
@@ -1,2 +1,5 @@
 test
- ,<fqsn,kfd<nklvdsnklsd<vnklvd<sknlvd<snklvdsnklvv sqnccklndsqnklcnk
\ No newline at end of file
+ ,<fqsn,kfd<nklvdsnklsd<vnklvd<sknlvd<snklvdsnklvv sqnccklndsqnklcnk
+
+
+ fqnjfnfdq
\ No newline at end of file
